.github/workflows/node.js.yml

```diff
@@ -22,7 +22,12 @@ jobs:
       uses: actions/setup-node@v2
       with:
         node-version: ${{ matrix.node-version }}
-    - run: npm ci
+    - run: npm install
+    - name: Archive package.lock
+      uses: actions/upload-artifact@v3
+      with:
+        name: package-lock.json
```

Comment:
While one could use this approach to debug, it still means you have to debug after it breaks, rather than have it not break unless you push an update.